### PR TITLE
update FE and some function missing

### PR DESCRIPTION
**Pull request by nampx-0860**

### CODE REVIEW

File: app/Http/Controllers/User/VoteController.php [hunk 1]
```diff
@@ -24,7 +24,9 @@ public function index()
         $pollInfo = Poll::where('id', $pollId)->first();
 
         $participantVote = ParticipantVote::with([
-            'participant',
+            'participant'=> function ($query) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 27</br>Expected 1 space between "'participant'" and double arrow; 0 found

File: app/Http/Controllers/User/VoteController.php [hunk 2]
```diff
@@ -24,7 +24,9 @@ public function index()
...
+                $query->with(['user']);
+            },
             'option',
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoKeySpecified</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 30</br>No key specified for array entry; first entry specifies key

File: app/Http/Controllers/User/VoteController.php (hunk 1 above)
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 27</br>Expected 1 space between "'participant'" and double arrow; 0 found

File: app/Http/Controllers/User/VoteController.php (hunk 2 above)
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoKeySpecified</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 30</br>No key specified for array entry; first entry specifies key

File: app/Http/Controllers/User/VoteController.php [hunk 3]
```diff
@@ -24,7 +24,9 @@ public function index()
         $pollInfo = Poll::where('id', $pollId)->first();
 
         $participantVote = ParticipantVote::with([
-            'participant',
+            'participant' => function ($query) {
+                $query->with(['user']);
+            },
             'option',
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoKeySpecified</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 30</br>No key specified for array entry; first entry specifies key

**Comment by nampx-0860** [comment]
updateFE

**Pull request merged**